### PR TITLE
Run tests in tox environment

### PR DESCRIPTION
**Pull request by Sniperik**
In this pull request, changes have been made to the way Python tests are run.
Previously, `coverage` was used with `xmlrunner` to run the tests.
Now, tests are being run within a `tox` environment using `pytest`.

### CODE REVIEW

**Comment by Sniperik** [comment]
Please make sure that [this pull request in common-ci](https://github.com/51Degrees/common-ci/pull/52) is merged prior to merging the current one. @justadreamer

File: fiftyone_devicedetection_cloud/tox.ini [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+description = run the tests with pytest
+package = wheel
+wheel_build_env = .pkg
+deps =
+    pytest>=6
+    pytest-cov
+    -e../fiftyone_devicedetection/
+commands =
+    pytest --cov=. --import-mode=append {tty:--color=yes} {posargs}
+;    - coverage combine
```
Comment (justadreamer) [inline fiftyone_devicedetection_cloud/tox.ini:15]: let's remove commented out lines

File: fiftyone_devicedetection_onpremise/tox.ini [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+wheel_build_env = .pkg
+deps =
+    pytest>=6
+    pytest-cov
+    Cython
+    flask
+commands =
+    python setup.py build_clib build_ext
+    pytest --cov=. --import-mode=append {tty:--color=yes} {posargs}
+;    - coverage combine
```
Comment (justadreamer) [inline fiftyone_devicedetection_onpremise/tox.ini:17]: and these too probably

**Review by justadreamer — APPROVED** [review]
approving, but let's remove unneeded commented lines

**Pull request merged**